src/Common/WKB.cpp

```diff
@@ -15,6 +16,26 @@ namespace ErrorCodes
 extern const int BAD_ARGUMENTS;
 }
 
+namespace
+{
+
+String getGeometricObjectTypeName(const GeometricObject & object)
+{
+    if (std::holds_alternative<CartesianPoint>(object))
+        return "Point";
+    else if (std::holds_alternative<LineString<CartesianPoint>>(object))
```

Comment:
For future PR (no need to change it now): It'd be better to have these classes declared somewhere `using CH_LineString = LineString<CartesianPoint>` or something similar, so as to not repeat them everywhere